backend/organization/serializers/organization.py

```diff
@@ -120,10 +120,12 @@ class OrganizationCardSerializer(serializers.ModelSerializer):
     types = serializers.SerializerMethodField()
     location = serializers.SerializerMethodField()
     name = serializers.SerializerMethodField()
+    hubs_count = serializers.SerializerMethodField()
     class Meta:
         model = Organization
         fields = (
-            'id', 'name', 'url_slug', 'thumbnail_image', 'location', 'types', 'short_description'
+            'id', 'name', 'url_slug', 'thumbnail_image', 'location', 'types', 'short_description', 'organization_size',
```

Comment:
Instead of  `organization_size`we should instead count the number of `OrganizationMember`s that are part of this organization

Reply:
Because organization_size is an optional field organization creators can fill out so not every organization has a value for that. Also it's quite motivating to see this number grow as more of the organization members join Climate Connect.

Reply:
Ah, sure. I didn't know that. It definitely makes sense :-)